types/oracledb/index.d.ts

```diff
@@ -3294,8 +3294,7 @@ declare namespace OracleDB {
          * @since 3.1
          */
         sessionCallback?:
-            | string
-            | ((connection: Connection, requestedTag: string, callback: (error?: DBError) => void) => void)
+            | ((connection: Connection, requestedTag: string, callback: (error?: unknown) => void) => void)
```

Comment:
Hi @sosoba, Any reason we are removing 'string' here?

Reply:
My mistake! Of course string (name of PL/SQL function) are allowed